hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -48,11 +48,13 @@ def __init__(self) -> None:
                                .format(dmem_size))
 
         # We represent the contents of DMEM as a list of 32-bit words (unlike
-        # the RTL, which uses a 256-bit array). An entry in self.data is None
-        # if the word has invalid integrity bits and we'll get an error if we
-        # try to read it. Otherwise, we store the integer value.
+        # the RTL, which uses a 256-bit array). An entry in self.data is a
```

Comment:
I know that this crappy text was by a certain Rupert Swarbrick, but... I think I probably meant "... RTL, which uses an array of 256-bit words." Since you're touching these lines anyway, would you mind tidying it up?